src/chaos-experiments/models/database/sequelize/sequelizeConnector.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+const KUBEOBJECT = 'kubeObject';
+let client;
+
+module.exports = {
+    init,
+    getAllChaosExperiments,
+    insertChaosExperiment,
+    getChaosExperimentById,
+    getChaosExperimentByName,
+    deleteChaosExperiment
+};
+
+async function init(sequelizeClient) {
+    client = sequelizeClient;
+    await initSchemas();
+}
+
+async function insertChaosExperiment(experimentId, experiment, contextId) {
+    const chaosExperimentModel = client.model('chaos-experiment');
```

Comment:
shouldnt the function generate the id?

Reply:
model generates it https://github.com/Zooz/predator/blob/chaos-experiments-api/src/chaos-experiments/models/chaosExperimentsManager.js#L18